components/datetime/src/any/zoned_datetime.rs

```diff
@@ -276,8 +276,8 @@ impl ZonedDateTimeFormatter {
     ///     TimeZoneFormatterOptions::default(),
     /// ).expect("Construction should succeed");
     ///
-    /// let (datetime, time_zone) = parse_zoned_gregorian_from_str("2021-04-08T16:12:37.000-07:00")
-    ///     .expect("Failed to parse zoned datetime");
+    /// let datetime = DateTime::new_gregorian_datetime(2021, 04, 08, 16, 12, 37).unwrap();
```

Comment:
Nit: Favor `new_iso_datetime` in AnyCalendar docs examples without arithmetic (see other thread)